docs/releases/1.11.txt

```diff
@@ -269,6 +269,10 @@ Tests
   :class:`~django.test.runner.DiscoverRunner` to make it easier to customize
   the :setting:`DEBUG` setting when running tests.
 
+* Added support for Python :meth:`python:unittest.TestCase.subTest`'s
```

Comment:
Chop "Python" and use a right quote in "'s" else sphinx renders it in the incorrect direction.